pkgs/by-name/fl/fluidsynth/package.nix

```diff
@@ -1,6 +1,10 @@
-{ stdenv, lib, fetchFromGitHub, buildPackages, pkg-config, cmake
-, alsa-lib, glib, libjack2, libsndfile, libpulseaudio
-, AppKit, AudioUnit, CoreAudio, CoreMIDI, CoreServices
+{ stdenv, darwin ? {}, lib, fetchFromGitHub, buildPackages, pkg-config, cmake,
+  alsa-lib, glib, libjack2, libsndfile, libpulseaudio,
+  AppKit    ? (if stdenv.hostPlatform.isDarwin then darwin.apple_sdk.frameworks.AppKit    else null),
```

Comment:
Normally you can just drop all Darwin frameworks, they are not needed after a recent SDK pattern change.